lib/Model/Attendee.php

```diff
@@ -87,6 +87,12 @@ class Attendee extends Entity {
 	public const PERMISSIONS_PUBLISH_VIDEO = 32;
 	public const PERMISSIONS_PUBLISH_SCREEN = 64;
 	public const PERMISSIONS_CHAT = 128;
+	public const PERMISSIONS_CHAT_REACTION = 256;
+	public const PERMISSIONS_FILE_VIEW = 512;
+	public const PERMISSIONS_FILE_SHARE = 1024;
```

Comment:
Should we do another for "create poll", or a generic one for "share (any thing but file) items"

Reply:
That's basically the second option. Since polls are highly more interactive I wondered if splitting makes sense, but yeah otherwise a generic object share also sounds like an option